### PR TITLE
Add support for smart switches

### PR DESCRIPTION
**Pull request by bdraco**
https://www.amazon.com/gp/product/B08PBN8R32/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

### CODE REVIEW

File: flux_led/__main__.py [hunk 1]
```diff
@@ -714,28 +728,24 @@ def update_state(self, retry=2):
         #     |  |  |  |  |  speed: 0x01 = highest 0x1f is lowest
         #     |  |  |  |  Mode WW(01), WW+CW(02), RGB(03), RGBW(04), RGBWW(05)
         #     |  |  |  preset pattern
-        #     |  |  off(23)/on(24)
+        #     |  |  off(23)/on(24) [bulbs], off(36)/on(35) [switches]
```
Comment (icemanch) [inline flux_led/__main__.py:731]: This is in Hex.  Can you double-check two things.

1. Is it returning 36/35 in Hex?
2. Double check that 36 is off and 35 is on. It's reversed from everything else they do. I wouldn't be surprised but, we should be sure.

**Comment by bdraco** [comment]
Power off state
`2021-10-02 04:39:51 DEBUG (SyncWorker_1) [flux_led.__main__] 192.168.106.147: new_state: 129,151,36,36,0,0,0,0,0,0,2,0,0,98,`

**Comment by bdraco** [comment]
Power on state
`2021-10-02 04:40:21 DEBUG (SyncWorker_3) [flux_led.__main__] 192.168.106.147: new_state: 129,151,35,35,0,0,0,0,0,0,2,0,0,96,`

File: flux_led/__main__.py (hunk 1 above)
Comment (bdraco) [reply flux_led/__main__.py:738]: My comments is in decimal.

```
        if power_state == 0x23:
            self._is_on = True
        elif power_state == 0x24:
            self._is_on = False
```

The existing inline comment is reverse.  The existing code below is correct.

I will fix the comment

`0x23` = `on`
`0x24` = `off`

Comment (bdraco) [reply flux_led/__main__.py]: [`cdc37eb` (#83)](https://github.com/Danielhiversen/flux_led/pull/83/commits/cdc37eb14e6ebf5d8276f86b06b187d828353a94)

**Review by icemanch — APPROVED** [review]

**Comment by icemanch** [comment]
I'm leaving this comment here as a reminder.  I'm going to merge this without the testing being completed but, I'll run the testing after the repo has been added to travisci.com.  The changes are low risk to final operation and testing was added by bdraco.

**Pull request merged**